Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnInit.hs

```diff
@@ -101,14 +101,20 @@ tfCustomer :: DInit.InitRes -> Maybe Spec.Customer
 tfCustomer res =
   return $
     Spec.Customer
-      { customerContact = Nothing,
+      { customerContact =
+          Just
+            Spec.Contact
+              { contactPhone = Just res.riderPhoneNumber
+              },
         customerPerson = do
-          riderName <- res.booking.riderName
-          Just $
-            Spec.Person
-              { personId = Nothing,
-                personImage = Nothing,
-                personName = Just riderName,
-                personTags = Nothing
-              }
+          if isJust res.riderName
```

Comment:
Why `<-` bind is changed to `if` condition? bind is more cleaner

Reply:
Using bind even if rider name is `Nothing` it will create an empty `Person`, changed to if to handle that condition. Maybe I could use the `>>=` operator to achieve the same